Source/HedgeModManager/Helpers.cs

```diff
@@ -27,6 +27,14 @@ public static string NormalizePath(string path)
         return path.Replace('\\', '/');
     }
 
+    /// <summary>
+    /// Normalizes a base URL path to end with /
+    /// </summary>
+    public static string NormalizeURL(string baseHost)
```

Comment:
I don't think `NormalizeURL` is a good name as it isn't really normalising, it is just converting to a directory path/ensuring the path ends with a slash. Can you please change it to something that explains the function better? I have provided one example below.

Edit: Might also be good to change the summary aswell. Something like
"Ensures that the specified path ends with a trailing forward slash (/)"

```suggestion
    public static string EnsureTrailingSlash(string baseHost)
```

Reply:
Changed to your suggestion, but kept "/" instead of (/) per other C# docs